app/Module.scala

```diff
@@ -53,22 +54,32 @@ class Module(val environment: Environment, val configuration: Configuration) ext
     bindConfigInt("mongodb.cache.expire.seconds", default = Some(60 * 30))
     bindConfigString("mongodb.name")
     bindConfigString("microservice.services.nps-hod.originatorId",       default = Some("HMRC_GDS"))
-    bindConfigString("microservice.services.nps-hod.path",               default = Some(""))
     bindConfigString("microservice.services.rti-hod.originatorId",       default = Some("local"))
     bindConfigString("microservice.services.rti-hod.env",                default = Some("local"))
     bindConfigString("microservice.services.rti-hod.authorizationToken", default = Some("local"))
     bindConfigBoolean("featureFlags.currentYearFlag")
     bindConfigBoolean("featureFlags.statePensionFlag")
     bindConfigBoolean("featureFlags.jobSeekersAllowanceFlag")
 
+    bindConfigProperty("des.authorizationToken")
```

Comment:
Is this "des.authorization" going to be the same as the "microservice.services.rti-hod.authorizationToken"?